src/components/Slider/__tests__/Slider.interactions.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from "@storybook/jest";
+import { within, userEvent } from "@storybook/testing-library";
+import { delay, interactionSuite, resetFocus } from "../../../__tests__/interactions-helper";
+
+// Decrease/Increase value by mouse click on Track/Rail of Slider
+const changeSliderValueByClickingOnTrackTest = async canvas => {
+  // prepare: take sizes of slider
+  const elRail = canvas.getByTestId("monday-slider-show-value-s__rail");
+  const rect = elRail.getBoundingClientRect();
+  // waiting for render
+  await delay(300);
```

Comment:
consider using `waitFor` (from `@testing-library/react`) along with `expect(element).toBeVisible()`, like we did on [`waitForElementVisible`](https://github.com/mondaycom/monday-ui-react-core/blob/master/src/__tests__/interactions-helper.js#L99)

Reply:
Changed to watiFor (i.e. to helper: waitForElementVisible from interactions-helper)